paper/myst.yml

```diff
@@ -41,14 +41,16 @@ project:
     # received: YYYY/MM/DD
     # accepted: YYYY/MM/DD
     published: 2022/01/20
-  keywords:
+  # keywords:
     # - KEYWORD 1
     # - KEYWORD 2
     # - KEYWORD 3
     # - KEYWORD 4
   abbreviations:
-    # CSEM: controlled source electromagnetics
-    # EM: electromagnetics
+    EM: Electromagnetics
+    LOI:  Loss On Ignition
+    PGI: Petrophysically and Geologically Guided Inversion
+    GMM: Gaussian Mixture Model
   thumbnail: thumbnail.png
```

Comment:
The thumbnail doesn't seem to want to show on this one. @stevejpurves: if you have a moment to take a look, any pointers are appreciated!!